src/west/manifest.py

```diff
@@ -278,7 +278,9 @@ def _load(self, data):
                                   revision=mp.get('revision'),
                                   west_commands=mp.get('west-commands'),
                                   remote=remote,
-                                  url=url)
+                                  url=url,
+                                  west_top=self_project.west_top,
+                                  )
```

Comment:
Style nit, please write:

```
west_top=self_project.west_top)
```

Reply:
Done but you understand why I did it this way, right?

Reply:
Trying to minimize diff in later patches?